scripts/release_scripts/update_configs.py

```diff
@@ -197,6 +197,11 @@ def add_redishost():
                 line = line.replace('localhost', redishost)
             f.write(line)
 
+    with python_utils.open_file(LOCAL_FECONF_PATH, 'r') as f:
```

Comment:
I don't quite understand what is going on here -- there's an additional check, but what exactly changed?

Reply:
Nothing changed, after updating the file there was no check to ensure that update is made. So, I just added this code in the update method itself to raise an error at the same point if update is not done.